build.gradle

```diff
@@ -86,6 +86,16 @@ task findScalaAndJavaTypes(type: Exec) {
     commandLine './scripts/checkScalaAndJavaFiles.sh'
 }
 
+
+task testWithDefaultReference(type: Test) {
```

Comment:
I'm not a huge fan of this but I don't see a really clever way to do it otherwise...